docs/project-setup/filesystem-sync.md

```diff
@@ -4,7 +4,7 @@ Outrigger uses [Unison](https://github.com/bcpierce00/unison) to provide high-pe
 
 With Unison, all the files from the root of your project are synced via a dedicated container and exposed as a named [Docker Volume](https://docs.docker.com/engine/tutorials/dockervolumes/) to the rest of your application. This volume is used as the file "source" for a volume mount to allow nearly native filesystem performance and features.
 
-For more on why Outrigger introduced Unison, jump down to [Why Sync Instead of NFS?](#why-sync-instead-of-nfs)
+For more on why Outrigger introduced unison, jump down to [Why Sync Instead of NFS?](#why-sync-instead-of-nfs)
```

Comment:
Should unison be capitalized or lower-case? Is our dictionary check case-sensitive? Should we fix this up in rig logging? (I belivee I recently capitalized this in a bunch of places.)

Reply:
It shoudl be capitalized unless we are referring directly to the binary, in which case it shoudl be lowercase and monospaced, `unison`